index.html

```diff
@@ -13,17 +13,27 @@
         <div class="content">
             <nav class="header">
                 <a class="logo" href="#">My Logo</a>
-                <a href="#" id="menu"><img src="img/menu.png" alt=""></a>
+                <a href="#" class="menu"><img src="img/menu.png"></a>
+                <a href="#" class="cancel"><img src="img/cancel2.png"></a>
+
                 <ul class="nav-links">
                     <li class="nav-link"><a href="#">Portfolio</a></li>
                     <li class="nav-link"><a href="#">About</a></li>
                     <li class="nav-link"><a href="#">Contact</a></li>
                 </ul>
+
+        
+              <ul class="mobile-links">
+                    <li class="nav-link"><a href="#cards" class="link">Portfolio</a></li>
+                    <li class="nav-link"><a href="#about-section" class="link2">About</a></li>
+                    <li class="nav-link"><a href="#contact-me" class="link3">Contact</a></li>
+              </ul>
```

Comment:
You did a good job working on this milestone! 💯 However, there's still some room to improve:

- [ ] Currently, your portfolio is having a horizontal overflow. Please consider fixing it! 🤓 🔧 

[Screencast from 27-06-23 10:26:30.webm](https://github.com/josman227/portfolio.github.io/assets/77069011/9b185cd8-c10a-45c6-bc85-3121ff516c57)

- [ ] Please consider following as closely as possible the given design:

|Template|Your design|
|---|---|
| ![image](https://github.com/josman227/portfolio.github.io/assets/77069011/2e59cf63-03df-45b0-9397-d3a7b430cb4c) | ![image](https://github.com/josman227/portfolio.github.io/assets/77069011/f1c8898b-025b-49b1-9cf3-64ca2f8c33ac) |

- 🚧 The background color is different. Please consider using `#6070FF`.
- 🚧 Your design has a vertical light line in the middle. Please consider removing it. 
- 🚧 The "X" icon should be white. Please fix it.